ui/components/app/edit-gas-fee-popover/network-status/tooltips.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import Tooltip from '../../../ui/tooltip';
+
+const NetworkStatusTooltip = ({ children, html, title }) => (
+  <Tooltip position="top" html={html} title={title} theme="tippy-tooltip-info">
+    {children}
+  </Tooltip>
+);
+
+NetworkStatusTooltip.propTypes = {
+  children: PropTypes.node.isRequired,
+  html: PropTypes.node,
+  title: PropTypes.string,
+};
+
+export const BaseFeeTooltip = ({ children }) => {
+  const t = useI18nContext();
+  return (
+    <NetworkStatusTooltip
+      html={t('networkStatusBaseFeeTooltip', [
+        <strong
+          key="base_fee_medium_estimate"
+          className="network-status__tooltip-label"
+        >
+          {t('medium')}
+        </strong>,
+        <strong
+          key="base_fee_high_estimate"
+          className="network-status__tooltip-label"
+        >
+          {t('high')}
+        </strong>,
+      ])}
+    >
+      {children}
+    </NetworkStatusTooltip>
+  );
+};
+
+BaseFeeTooltip.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+export const PriorityFeeTooltip = ({ children }) => {
+  const t = useI18nContext();
+  return (
+    <NetworkStatusTooltip title={t('networkStatusPriorityFeeTooltip')}>
+      {children}
+    </NetworkStatusTooltip>
+  );
+};
+
+PriorityFeeTooltip.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+export const NetworkStabilityTooltip = ({ children, statusInfo }) => {
+  const t = useI18nContext();
+
+  return (
+    <NetworkStatusTooltip
+      html={t('networkStatusStabilityFeeTooltip', [
+        <strong
+          key="network-status__tooltip"
+          className="network-status__tooltip-label"
+          style={{ color: statusInfo?.color }}
+        >
+          {t(statusInfo?.tooltipLabel)}
```

Comment:
`statusInfo?.tooltipLabel` could be null — is that what we want?

Also it looks like this will appear as capitalized based on your screenshot. What do we feel about using a lowercased version of the status?

Reply:
That check for null was not needed I removed it, I fixed the case issue bu adding corresponding lowercase i18n values.